src/views/Color/Color.tsx

```diff
@@ -42,16 +47,26 @@ const ColorSubComponent = ({ color, ...props }: ColorSubComponentProps) => {
   );
 };
 
-export const Color: ColorView = ({
-  red = 0,
-  green = 0,
-  blue = 0,
-  ...props
-}) => {
+const getColor = (props: ColorProps) => {
```

Comment:
This looks awesome. This is how I would refactor this. 

```jsx
const getColorFromProps = (props: ColorProps) => {
  if (props.color) return props.color;
  else if (props.red || props.green || props.blue) {
    return`rgb(${props.red || 0},${props.green || 0},${props.blue || 0})`;
  }
  return  "primary";
 
}

export const Color: ColorView = (props: ColorProps) => {
  return (
    <Rectangle
      frame={{ width: 100, height: 100 }}
      fill={getColorFromProps(props)}
      {...props}
    />
  );
```

Reply:
do we need to remove keys from props since we're no longer destructuring?

Reply:
What do you mean by keys?

Reply:
also, I think in `getColorFromProps` we need to key in, `props['color']`, to get values because of the or'ed part of the `ColorProps`.

Reply:
> What do you mean by keys?

```
  fill={getColorFromProps(props)}
  {...props} // this has red/blue/green/color still in it
```

Reply:
ah in that case use the `in` operator 
`if ('color' in props)`

Reply:
and to the point about keys, yes youre right. we will need to pass the rest of the props without those somehow

Reply:
I like this approach. It's not the most syntactic sugar but it's clear and would be less complexity to manage. 

Will update with this soon. Any other items before we merge?

Reply:
dont think so, just gotta do some testing!